src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,71 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { FC } from 'react';
+import { getPagesCounter } from '../../utils/pagination/getPagesCounter';
+import { getTotalPage } from '../../utils/pagination/getTotalPage';
+
+interface IPagination {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<IPagination> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = getTotalPage(total, perPage);
+  const pageCounter = getPagesCounter(totalPages);
+
+  const prevPageHandler = () =>
+    currentPage - 1 < 1 ? undefined : onPageChange(currentPage - 1);
+
+  const nextPageHandler = () =>
+    currentPage + 1 > totalPages ? undefined : onPageChange(currentPage + 1);
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href={`#${currentPage}`}
```

Comment:
Using `href` with `#${currentPage}` can cause the page to jump to the top when clicked. Consider using `event.preventDefault()` in the `prevPageHandler` to prevent this behavior.